network/src/prover.rs

```diff
@@ -203,13 +207,13 @@ impl<N: Network, E: Environment> Prover<N, E> {
     ///
     /// Sends a `PoolRegister` message to the pool IP address.
     ///
-    async fn send_pool_register(&self) {
+    pub async fn send_pool_register(&self) {
```

Comment:
Why is this `pub`? These methods are meant to be internal, otherwise there is "crossing of wires" which can lead to unintentional node halts